pages.fa/common/7z.md

```diff
@@ -0,0 +1,36 @@
+# 7z
+
+> آرشیو کننده فایل با نسبت فشرده سازی بالا.
+> اطلاعات بیشتر: <https://www.7-zip.org>.
+
+- آرشیو کننده فایل یا دایرکتوری:
+
+`7z a {{path/to/archive.7z}} {{path/to/file_or_directory}}`
```

Comment:
Please translate the placeholders, like `path/to/file_or_directory` for get this easier for non-English speakers.